Documentation/gettingstarted/hubble-enable.rst

```diff
@@ -0,0 +1,41 @@
+Enable Hubble
+==============
+
+Hubble is a fully distributed networking and security observability platform
+for cloud native workloads. It is built on top of Cilium and eBPF to enable
+deep visibility into the communication and behavior of services as well as the
+networking infrastructure in a completely transparent manner.
+
+Deploy Hubble using Helm:
+
+.. parsed-literal::
+
+   helm upgrade cilium |CHART_RELEASE| \\
+      --namespace kube-system \\
+      --set global.hubble.enabled=true \\
+      --set global.hubble.listenAddress=":4244" \\
+      --set global.hubble.metricsServer=":54321" \\
```

Comment:
not sure what a good default port is here.

Reply:
I don't know either, but right now Cilium exposes metrics on the following ports:
- 9090 cilium
- 6942 cilium-operator

I think 9091 which we use in CI might be a decent choice. I wonder if it maybe makes sense to rewrite the helm charts so we always have a default port and users only need to set `global.hubble.metrics.enabled`.

Reply:
PR to simplify the metric values: https://github.com/cilium/cilium/pull/11887

Reply:
rebased to pick up #11887